src/redirects.json

```diff
@@ -461,7 +461,8 @@
     "/resources/guide-to-user-data-security": "/articles/security/guide-to-user-data-security",
     "/upgrade/from-homegrown": "/compare",
     "/upgrade/from-open-source": "/compare",
-    "/upgrade/from-saas": "/compare"
+    "/upgrade/from-saas": "/compare",
+    "/getit": "https://raw.githubusercontent.com/FusionAuth/fusionauth-install/master/getit.sh"
```

Comment:
Minor nitpick -- the rest of this list is in alpha order. This should be, too.

Reply:
Yup, done.

Just a note: the current file has some inconsistencies RE the order. I ran it through `sort` and it has 45 lines that should be reordered.